PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1837,6 +1837,58 @@ bool doesPathExist(const wchar_t* path, DWORD milliSec2wait, bool* isNetWorkProb
 	return (attr != INVALID_FILE_ATTRIBUTES);
 }
 
+
+//----------------------------------------------------
+
+struct wfopenParamResult
+{
+	std::wstring _filePath;
+	FILE* _pFile = nullptr;
+	bool _isNetworkFailure = true;
+	wfopenParamResult(wstring filePath) : _filePath(filePath) {};
+};
+
+DWORD WINAPI wfopenWorker(void* data)
+{
+	wfopenParamResult* inAndOut = static_cast<wfopenParamResult*>(data);
+	inAndOut->_pFile = _wfopen(inAndOut->_filePath.c_str(), L"rb");
+	inAndOut->_isNetworkFailure = false;
+	return ERROR_SUCCESS;
+};
+
+FILE* wfopenWaitSec(const wchar_t* filePath, DWORD milliSec2wait, bool* isNetWorkProblem)
+{
+	wfopenParamResult data(filePath);
+
+	HANDLE hThread = ::CreateThread(NULL, 0, wfopenWorker, &data, 0, NULL);
+	if (!hThread)
+	{
+		return FALSE;

```

Comment:
rather `return nullptr;`

even though the `FALSE` is also numerically ok here

Reply:
Well spotted!